samples/word/25-paragraph/get-word-count.yaml

```
@@ -12,11 +12,13 @@ script:
         $("#count").click(() => tryCatch(run));
 
         async function run() {
+          // Count how many times each word/term appears in the document.
```

Suggestion to avoid the '/'. "Term" should be inclusive of "word".
```suggestion
          // Count how many times each term appears in the document.
```